sdk/purview/purview-workflow-rest/src/responses.ts

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { HttpResponse } from "@azure-rest/core-client";
+import {
+  WorkflowMetadataListOutput,
+  ErrorResponseOutput,
+  WorkflowOutput,
+  UserRequestResponseOutput,
+  WorkflowRunListOutput,
+  WorkflowRunOutput,
+  TasksListOutput,
+  WorkflowTaskOutput,
+} from "./outputModels";
+
+/** List all workflows. */
+export interface ListWorkflows200Response extends HttpResponse {
+  status: "200";
+  body: WorkflowMetadataListOutput;
+}
+
+/** List all workflows. */
+export interface ListWorkflowsDefaultResponse extends HttpResponse {
+  status: string;
+  body: ErrorResponseOutput;
+}
+
+/** Get a specific workflow. */
+export interface GetWorkflow200Response extends HttpResponse {
+  status: "200";
+  body: WorkflowOutput;
+}
+
+/** Get a specific workflow. */
+export interface GetWorkflowDefaultResponse extends HttpResponse {
+  status: string;
+  body: ErrorResponseOutput;
+}
+
+/** Create or replace a workflow. */
+export interface CreateOrReplaceWorkflow200Response extends HttpResponse {
+  status: "200";
+  body: WorkflowOutput;
+}
+
+/** Create or replace a workflow. */
+export interface CreateOrReplaceWorkflowDefaultResponse extends HttpResponse {
+  status: string;
+  body: ErrorResponseOutput;
+}
+
+/** Delete a workflow. */
+export interface DeleteWorkflow204Response extends HttpResponse {
+  status: "204";
+  body: Record<string, unknown>;
```

Comment:
```suggestion
```
@joheredi should we remove it before the codegen fix this issue https://github.com/Azure/autorest.typescript/issues/1771 ?

Reply:
@CordeliaGrey please commit this change suggestion

Reply:
Fixed, thanks.